doc/self-hosted/index.md

```diff
@@ -115,5 +115,30 @@ OPTIONS
   ```
   *Important:* For more details about Apache reverse proxy configuration checkout the [documentation](https://httpd.apache.org/docs/current/mod/mod_proxy.html) - especially the [Securing your Server](https://httpd.apache.org/docs/current/mod/mod_proxy.html#access) section
   
+  ### Start code-server as systemd unit
+  Create `/lib/systemd/system/code-server.service` with the following contents:
+  ```
+  [Unit]
+  Description=Code Server IDE
+  After=network.target
+
+  [Service]
+  Type=simple
+  User=<USER>
+  EnvironmentFile=$HOME/.profile
+  Restart=on-failure
+  RestartSec=10
+
+  ExecStart=<PATH TO BINARY> --allow-http
+
+  StandardOutput=file:/var/log/code-server-output.log
```

Comment:
On Ubuntu 16.04 the Standard* options were not recognized. I ended up with the following systemd unit (I anyways prefer logging to go to journald):

```
[Unit]
Description=VSCode in a browser
After=network.target

[Service]
Type=simple
User=myuser
EnvironmentFile=$HOME/.profile
Restart=on-failure
RestartSec=10

ExecStart=/usr/local/bin/code-server --allow-http --port 1234 --no-auth --disable-telemetry $HOME/dev-workspace

ExecStop=/sbin/start-stop-daemon --stop -x /usr/local/bin/code-server

[Install]
WantedBy=multi-user.target
```

Reply:
Have you tested this on other distros? This should work on all that uses systemd not just one

Reply:
I think a more viable alternative is to log this to journald if needed.

Reply:
We should always be logging to journald.